larray/tests/test_excel.py

```diff
@@ -80,11 +80,6 @@ def test_setitem(self):
             assert wb.sheet_names() == ['sheet1', 'sheet2', 'sheet3']
             assert wb['sheet2']['A1'].value == 'sheet1 content'
 
-            with open_excel(visible=False, app="new") as wb2:
```

Comment:
Please keep this test intact. It **must** raise. It should be fixed in master (commit c8aac0b576599b8ba6dabddbb0a760b7fa3d49ac). I guess you haven't rebased yet.